packages/modular-scripts/src/__tests__/addPackage.test.ts

```diff
@@ -2,91 +2,84 @@ import execa from 'execa';
 import rimraf from 'rimraf';
 import path from 'path';
 import fs from 'fs-extra';
+import tree from 'tree-view-for-tests';
 
 import getModularRoot from '../utils/getModularRoot';
 
 import type { CoreProperties } from '@schemastore/package';
+import { createModularTestContext, mockInstallTemplates } from '../test/utils';
 
 const modularRoot = getModularRoot();
 
-// These tests must be executed sequentially with `--runInBand`.
+// Setup temporary test context
+const tempModularRepo = createModularTestContext();
+console.log(`Temp Modular Repo: ${tempModularRepo}`);
+if (fs.existsSync(tempModularRepo)) {
+  console.log('Exists');
+}
+const packagesPath = path.join(tempModularRepo, 'packages');
+
+const templatesPath = path.join(modularRoot, '__fixtures__', 'templates');
+mockInstallTemplates(templatesPath, tempModularRepo);
 
-const packagesPath = path.join(getModularRoot(), 'packages');
+/**
+ * Reset 'packages' workspace in temporary Modular Repo
+ */
+function cleanup() {
+  rimraf.sync(packagesPath);
+  fs.mkdirSync(packagesPath);
+}
 
 function modular(str: string, opts: Record<string, unknown> = {}) {
-  return execa('yarnpkg', ['modular', ...str.split(' ')], {
-    cwd: modularRoot,
+  return execa('yarnpkg', ['modular', ...str.split(' '), '--verbose'], {
```

Comment:
Commenting to remind us to remove this at clean-up

Reply:
Thanks!

Reply:
Thanks, done